docs/sphinx/source/whatsnew/v0.11.3.rst

```diff
@@ -12,6 +12,8 @@ Breaking Changes
 * Users must now provide ModelChain.spectral_model, or the 'no_loss' spectral
   model is assumed. pvlib.modelchain.ModelChain no longer attempts to infer
   the spectral model from PVSystem attributes. (:issue:`2017`, :pull:`2253`)
+* Remove deprecated :py:func:`pvlib.pvsystem.sapm_spectral_loss`.
```

Comment:
```suggestion
* Remove deprecated :py:func:`!pvlib.pvsystem.sapm_spectral_loss`.
```

Reply:
Curious - what does the "!" do?

Reply:
Aha https://github.com/pvlib/pvlib-python/pull/2409#discussion_r1996113760